frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -1704,7 +1704,11 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
                             ...iTransformCellProps,
                           }
                           return (
-                            <Td key={`row-${rowIndex}-cell-${cellIndex}`} {...rowProps} isActionCell={isActionKebab}>
+                            <Td
+                              key={`cell-${row.props.key}-${selectedSortedCols[cellIndex]?.header}`}
+                              {...rowProps}
```

Comment:
Best option I found the row cells was the using the row key with the column header.